cleaners/miro.xml

```diff
@@ -18,26 +18,48 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Miro
+    @url http://www.getmiro.com/
+    @os Windows, Linux
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-14
+    @cleanerby ??? & https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows XP
+    @testeddate 2019-03-12 (... with an old version of the cleaner!)
+    @testedby https://github.com/bleachbit/bleachbit/issues/460#issuecomment-471987298
+    @note 
+
 -->
 <cleaner id="miro">
   <label>Miro</label>
   <description>Multimedia viewer</description>
+  <var name="cache-profile">
+    <value search="glob" os="windows">$AppData\Participatory Culture Foundation\Miro\Support</value>
+    <value search="glob" os="linux">~/.miro</value>
+  </var>
   <option id="cache">
     <label>Cache</label>
     <description>Delete the cache</description>
-    <action command="delete" search="walk.files" path="~/.miro/icon-cache/"/>
-    <action command="delete" search="walk.files" path="~/.miro/mozilla/Cache/"/>
-    <action command="delete" search="walk.files" path="$APPDATA\Participatory Culture Foundation\Miro\Support\icon-cache\"/>
+    <action command="delete" search="walk.files" path="$$cache-profile$$/icon-cache/"/>
+    <action command="delete" search="walk.files" path="$$cache-profile$$/mozilla/Cache/"/>
   </option>
   <option id="logs">
     <label>Logs</label>
     <description>Delete the logs</description>
     <action command="delete" search="file" path="~/.miro/miro-downloader-log"/>
     <action command="delete" search="file" path="~/.miro/miro-log"/>
     <action command="delete" search="file" path="~/.miro/miro-log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro-downloader.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro_Download_Daemon_a.txt"/>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
+    <action command="delete" search="file" path="$Temp\Miro_Download_Daemon_a.txt"/>
```

Comment:
Line 55 deletes the same file (Miro_Download_Daemon_a.txt) deleted on line 63. Would you agree line 55 should be deleted?

Reply:
I would let it...